urbanbroccoli/gaussseidel.py

```diff
@@ -74,12 +74,39 @@ def gaussseidel_3d(rho, hx, hy, hz, epsilon, maxiter, maxerr):
     return phi
 
 def gaussseidel(rho, h, epsilon=1.0, maxiter=10000, maxerr=1e-15):
+    '''
+    Gauss-Seidel relaxation method to solve a linear system of equation
+    =====
```

Comment:
Why this line?

Reply:
Esthetic? No particular purpose, I can take it out...